pageserver/src/pgdatadir_mapping.rs

```diff
@@ -633,7 +633,7 @@ impl Timeline {
 
     pub(crate) async fn get_twophase_file(
         &self,
-        xid: TransactionId,
+        xid: u64,
```

Comment:
If the length of TransactionId changed, let's update our types or add new ones, rather than using a less-specifically-typed u64